osu.Game/Overlays/DialogOverlay.cs

```diff
@@ -88,16 +94,11 @@ protected override void PopIn()
         protected override void PopOut()
         {
             base.PopOut();
+            this.FadeOut(PopupDialog.EXIT_DURATION, Easing.InSine);
```

Comment:
Is this reordering intended? Previously the overlay would only fade out if the current dialog was present and visible. May be fine, mostly just wondering.

Reply:
I actually don't know. The `IsPresent` override on this class makes fading a bit of a mystery.

In testing removing both the fade in and out seems to have zero effect, as that is managed by the individual popups. So I'll just remove both while we're here and see how that plays out.